presto-docs/src/main/sphinx/release/release-321.rst

```diff
@@ -0,0 +1,61 @@
+===========
+Release 321
+===========
+
+General Changes
+---------------
+
+* Fix incorrect result of ``round`` function when applied to ``tinyint``, ``smallint``, ``integer``
+  or ``bigint`` type with negative decimal places. (:issue:`42`)
+* Load data lazily in connector ``information_schema`` tables. This highly improves
+  performance for queries like: ``SELECT ... FROM catalog.information_schema.columns ... LIMIT ...```. (:issue:`1543`)
+
+Security Changes
+----------------
+
+* Improve query results authorization security with one-time tokens. (:issue:`1654`)
+
+Hive Connector Changes
+----------------------
+
+* Fix reading ``TEXT`` file collection delimiter set by Hive version earlier
+  than 3.0. (:issue:`1714`)
+* Fix AWS Glue support broken in Presto 320. (:issue:`1698`)
+* Improve performance for broadcast joins by using dynamic filtering.
```

Comment:
Move this to General since it is an engine change. We can add a note here like "Add support for dynamic filtering for broadcast joins"